boardswarm/src/pdudaemon.rs

```diff
@@ -97,17 +97,17 @@ impl crate::Actuator for PduDaemonActuator {
     async fn set_mode(
         &self,
         parameters: Box<dyn erased_serde::Deserializer<'static> + Send>,
-    ) -> Result<(), crate::ActuatorError> {
+    ) -> Result<(), ActuatorError> {
         #[derive(Deserialize)]
         struct ModeParameters {
             mode: String,
         }
         let parameters = ModeParameters::deserialize(parameters).unwrap();
         match parameters.mode.as_str() {
-            "on" => self.daemon.on(&self.hostname, &self.port).await.unwrap(),
-            "off" => self.daemon.off(&self.hostname, &self.port).await.unwrap(),
+            "on" => self.daemon.on(&self.hostname, &self.port).await,
+            "off" => self.daemon.off(&self.hostname, &self.port).await,
             _ => todo!(),
         }
-        Ok(())
+        .map_err(|_e| ActuatorError {})
```

Comment:
Not sure if this is the right thing to do, possibly we could return the actual error from the daemon. But it currently works!

Reply:
Ideally the errors from actuators should be more descriptive but this still is a net improvement :)

Reply:
I created https://github.com/boardswarm/boardswarm/issues/92 to handle that. I doubt I will have the time to do that here :-).